### PR TITLE
Registration API signed auth header examples.

### PR DESCRIPTION
**Pull request by tcmitche**

### CODE REVIEW

File: signed-header-requests/signed-request.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+const crypto = require('crypto');
+
+function makeSignedAuthHeader(endpoint, params, datetime, client_id, secret) {
+  const hmac = crypto.createHmac('sha1', secret)
```
Comment (rdenmark) [inline signed-header-requests/signed-request.js]: Moving the declaration of `hmac` to just before its usage would keep this example consistent with the others.

Comment (tcmitche) [inline signed-header-requests/signed-request.js]: Ah good point!

**Comment by pdedmon** [comment]
The python and js look fine to me. From what I can tell, the ruby and php are consistent.

**Comment by Jared-Janrain** [comment]
👍

**Pull request merged**